docs/error-messages/compiler-warnings/compiler-warning-levels-3-and-4-c4244.md

```diff
@@ -40,12 +39,15 @@ For more information, see [Usual Arithmetic Conversions](../../c-language/usual-
 // C4244_level3.cpp
 // compile with: /W3
 int main() {
-   __int64 i = 8;
-   unsigned int ii = i;   // C4244
+   __int64 i64 = 1;
+   unsigned __int64 u64 = 2;
+
+   int int1 = i64;   // C4244
+   int int3 = u64;   // C4244
 }
 ```
 
-Warning C4244 can occur when building code for 64-bit targets that does not generate the warning when building for 32-bit targets. For example, a difference between pointers is a 32-bit quantity on 32-bit platforms, but a 64-bit quantity on 64-bit platforms.
+Warning C4244 can occur when building code for 64-bit targets that does not generate the warning when building for 32-bit targets. For example, a pointer is a 32-bit quantity on 32-bit platforms, but a 64-bit quantity on 64-bit platforms.
```

Comment:
The meaning was correct before - it's talking about the result of pointer arithmetic. Please put the original text back.

Reply:
Oh, did not realize that was referring to the difference as a result of a pointer arithmetic. Thought it was a just a general use of the word 'difference'.